os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -219,19 +220,22 @@ static int binary_manager_load(int bin_idx)
 #ifdef CONFIG_SUPPORT_COMMON_BINARY
 			if (bin_idx == BM_CMNLIB_IDX) {
 				ret = binary_manager_read_header(BINARY_COMMON, devpath, &common_header_data, true);
+				BIN_VER(bin_idx, BIN_USEIDX(bin_idx)) = common_header_data.version;
 			} else
 #endif
 			{
 				ret = binary_manager_read_header(BINARY_USERAPP, devpath, &user_header_data, true);
+				BIN_VER(bin_idx, BIN_USEIDX(bin_idx)) = user_header_data.version;
```

Comment:
`common_header_data` has a member of `version`, but `user_header_data` has a `bin_ver`.
How about unifying the member variable name?

Reply:
I did to distinguish  2 version of kernel and user, `bin_ver` and `kernel_ver` in user binary header.